### ISSUE TITLE
JS error in kv-tree.js

### ISSUE THREAD

**Issue by rimas-kudelis** [kartik-v/yii2-tree-manager#78]
I get the following JS error in `TreeViewInput`, unless I set `showTooltips` to `false`:

```
TypeError: self.$toolbar.find(...).tooltip is not a function kv-tree.js:97:17
```

**Comment by kartik-v** [comment]
Tooltips uses Bootstrap Tooltips and you need `bootstrap.js` loaded on your page for it to work (part of the `yii\bootstrap\BootstrapPluginAsset`) 

The extension does not load Bootstrap.js assets - and leaves this to each developer to use it based on their need.

**Comment by rimas-kudelis** [comment]
Why doesn't TreeViewAsset depend on BootstrapPluginAsset in that case? Seems logical to me that it should, because why would the user have to track dependencies manually if that can be done automatically?..

**Comment by kartik-v** [comment]
Because this is an optional requirement. There are many who do not wish to load bootstrap.js in their application for performance or app specific issues.

**Comment by rimas-kudelis** [comment]
Shouldn't it be mentioned in the docs near each affected option at least?
E.g.:

> `asDropdown`: boolean, whether to display the tree view input widget as a dropdown. Defaults to `true`. If set to false, the tree view input widget will be embedded inline. **NB:** Make sure to load BootstrapPluginAsset if this is `true`!

Because right now, I did everything by the docs, and the result didn't work. Wanna guess whom or what I blamed? :)

By the way, you're saying this is an optional requirement, but `asDropdown` is `true` **by default**, thus rendering the asset required by default. Such policy doesn't make sense, at least to me.

Perhaps there is a way to automatically declare the asset as required when the affected options are used? That would be ideal.

**Comment by kartik-v** [comment]
Yes will update the docs. Maybe an exception can be thrown as well.

**Comment by kartik-v** [comment]
Noting an enhancement to do this better.

**Comment by rimas-kudelis** [comment]
So is there a way to declare a dependency dynamically?..
Also, you should probably change the tag of this issue. ;)

**Comment by kartik-v** [comment]
Fixed via 4bec380.

**Comment by rimas-kudelis** [comment]
Thank you!
